shared/tasks/styles.mjs

```diff
@@ -79,10 +94,8 @@ export async function compileStylesheet ([modulePath, { srcPath, destPath, fileP
     }))
 
     // Pass source maps to PostCSS
-    options.map = {
-      annotation: true,
-      inline: false,
-      prev: map
+    if (typeof options.map === 'object') {
+      options.map.prev = map
```

Comment:
PostCSS will automatically discover source maps via `sourceMappingURL` comments but we continue to pass them directly from Dart Sass here